src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs

```diff
@@ -179,6 +179,8 @@ static async Task CallJSMethod(FrameworkElement platformWebView, string script)
 	{
 		if (platformWebView is WebView2 webView2)
 		{
+			await webView2.EnsureCoreWebView2Async();
```

Comment:
we only need to call it once. lets move it to CreatePlatformView

Reply:
It should be  awaited. Moving to CreatePlatformView would require modification of that method to async. 
Was unsure of side effects.

In the remarks for the call.... 
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.wpf.webview2.ensurecorewebview2async

there is no issue calling this mutiple times, and potentially a change in the view might need to call this again, therefore I placed it here.

Thoughts?

Reply:
If there's no issue calling `EnsureCoreWebView2Async` multiple times, I'm cool with keeping it in `CallJSMethod`. 

We could call `EnsureCoreWebView2Async` in `CreatePlatformView()`, but we'd need to include a fire-and-forget `async void`, and I prefer to avoid this given that calling `EnsureCoreWebView2Async` from `CallJSMethod` works. 

Just for fun, here's how we would modify `CreatePlatformView`:

```cs
	protected override FrameworkElement CreatePlatformView()
	{
		if (string.IsNullOrEmpty(MapsKey))
		{
			throw new InvalidOperationException("You need to specify a Bing Maps Key");
		}

		var mapPage = GetMapHtmlPage(MapsKey);
		var webView = new MauiWebView(new WebViewHandler());
		webView.NavigationCompleted += HandleWebViewNavigationCompleted;
		webView.WebMessageReceived += WebViewWebMessageReceived;
		webView.LoadHtml(mapPage, null);

		InitializeMap(webView);

		return webView;

		// Use async-void to properly use async/await to initialize the map from a non-async method
		static async void InitializeMap(MauiWebView webView2)
		{
			await webView2.EnsureCoreWebView2Async();
		}
	}
```